### PR TITLE
add embedding_lookup doc

### PR DESCRIPTION
**Pull request by funsimple**
# Description

add embedding_lookup doc

### CODE REVIEW

**Comment by funsimple** [comment]
@googlebot I fixed it.

**Comment by funsimple** [comment]
@googlebot I fixed it.

File: docs/basic_ranking.ipynb [hunk 1]
```diff
@@ -0,0 +1,302 @@
+{
```
Comment (rhdong) [inline docs/basic_ranking.ipynb:1]: please change filename to basic_ranking_with_dynamic_embedding.ipynb

**Review by rhdong — APPROVED** [review]
LGTM

**Pull request merged**